spec/services/stats/report_generator_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.shared_examples 'package stats in CSV' do
```

Comment:
I think it would be clearer to name it something like this

```suggestion
RSpec.shared_examples 'a stats report CSV exporter' do
```